Classes and objects/__str__ vs __repr__/tests.py

```diff
@@ -0,0 +1,41 @@
+from test_helper import run_common_tests, get_answer_placeholders, passed, failed
+
+
+def test_window():
```

Comment:
This is probably for a future modification, but we really should think of providing meaningful tests' names